packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -161,34 +152,7 @@ export async function _receiveCloudEventsDeserialize(
     throw createRestError(result);
   }
 
-  return {
-    value: result.body["value"].map((p: any) => {
-      return {
-        brokerProperties: {
-          lockToken: p.brokerProperties["lockToken"],
-          deliveryCount: p.brokerProperties["deliveryCount"],
-        },
-        event: {
-          id: p.event["id"],
-          source: p.event["source"],
-          data: p.event["data"],
-          dataBase64:
-            typeof p.event["data_base64"] === "string"
-              ? stringToUint8Array(p.event["data_base64"], "base64")
-              : p.event["data_base64"],
-          type: p.event["type"],
-          time:
-            p.event["time"] !== undefined
-              ? new Date(p.event["time"])
-              : undefined,
-          specversion: p.event["specversion"],
-          dataschema: p.event["dataschema"],
-          datacontenttype: p.event["datacontenttype"],
-          subject: p.event["subject"],
-        },
-      };
-    }),
-  };
+  return result.body;
```

Comment:
the same here, the rest api layer name is different from client layer, we should have deserializer for it.